README.md

```diff
@@ -95,7 +95,7 @@ app][electron].
 
 * **Glue code**. We maintain a [Hubot adapter][hubot-adapter] and several
 integrations ([Phabricator][phab], [Jenkins][], [Puppet][], [Redmine][],
-and [Trello][]), plus [node.js API bindings][node], and a [full-text search
+and [Trello][]), plus [node.js API bindings][node], an [isomorphic javascript library][javascript], and a [full-text search
```

Comment:
The docs in the readme at https://github.com/zulip/zulip-node seem to indicate that the only supported api operation is sending a message, which can indeed be done with zulip-js.